src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  peopleList: Person[],
+  selectedPerson: string | null,
+  isNoPeople: boolean,
+};
+
+export const PeopleTable: React.FC<Props> = (
+  { peopleList, selectedPerson, isNoPeople },
+) => {
+  const isSelected = (person: Person) => person.slug === selectedPerson;
+
+  const getSlug = (name: string | null) => {
+    const findPerson = peopleList.find(person => person.name === name);
+
+    return findPerson?.slug;
+  };
+
+  return (
+    <>
+      {peopleList.length > 0 && (
+        <table
+          data-cy="peopleTable"
+          className="table
+            is-striped
+            is-hoverable
+            is-narrow
+            is-fullwidth
+          "
+        >
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {peopleList.map(person => (
+              <tr
+                data-cy="person"
+                key={person.born}
```

Comment:
I have such message...
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.
SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.5.0
YOUR TYPESCRIPT VERSION: 4.9.5

But actually I have 4.4.3 typescript version.🤷‍♂️

Reply:
Does this problem appear on using `person.id`?
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster because your answers in GitHub will be visible to the mentor only after you re-request review 🙂

Reply:
No, this message appears when I run npm lint. It's like a glitch🙂

Reply:
Ok. Thanks for advice!